internal/scheduler/scheduler.go

```diff
@@ -0,0 +1,81 @@
+package scheduler
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/robfig/cron/v3"
+	"github.com/techbloghub/server/config"
+	"github.com/techbloghub/server/ent"
+	"github.com/techbloghub/server/internal/rss"
+)
+
+// Scheduler manages all scheduled tasks
+type Scheduler struct {
+	client *ent.Client
+	cron   *cron.Cron
+	config *config.SchedulerConfig
+}
+
+// NewScheduler creates a new scheduler
+func NewScheduler(client *ent.Client, config *config.SchedulerConfig) *Scheduler {
+	// Create a new cron scheduler with seconds field enabled
+	c := cron.New(cron.WithSeconds())
+	return &Scheduler{
+		client: client,
+		cron:   c,
+		config: config,
+	}
```

Comment:
```suggestion
        c := cron.New(
          cron.WithSeconds(),
          cron.WithLogger(cron.VerbosePrintfLogger(log.Default())), // 상세히 찍히는 로그!
          cron.WithChain(
            cron.Recover(cron.DefaultLogger), // panic 복구
          ),
        )
        return &Scheduler{
		client: client,
		cron:   c,
		config: config,
	}
```

찾아보니 요래 미들웨어형식으로 추가해줄 수 있다고 하네
withChain은 Job 돌아가다가 Panic이 발생해도 스케줄러는 계속 돌아가도록 하는 넘!